serverlessworkflow/modules/ROOT/pages/tooling/quarkus-dev-ui-extension/quarkus-dev-ui-overview.adoc

```diff
@@ -0,0 +1,45 @@
+= Kogito Serverless Workflow Tools - Quarkus Dev UI
+
+Quarkus applications comes with an interactive Dev UI, that provides developers with a number of tools to help in the process of development. You can plug in these tools to a Quarkus application in the form of extensions. The Kogito Serverless Workflow Tools is an extension, that provides a Console for you to view, manage and start Workflows Instances. You can add this extension as a _dependency_ to the _pom.xml_ of your Kogito Project.
+
+.Kogito Serverless Workflow Tools dependency in pom.xml
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-quarkus-serverless-workflow-devui</artifactId>
+  <version>${VERSION}</version>
+</dependency>
+----
+
+Another dependency for the Console is to add the _kogito-addons-quarkus-process-svg_ to provide the SVG diagram to the Consoles.
+
+.Quarkus Addons:Process-SVG dependency in pom.xml
+[source,xml]
+----
+<dependency>
+  <groupId>org.kie.kogito</groupId>
+  <artifactId>kogito-addons-quarkus-process-svg</artifactId>
+</dependency>
+----
```

Comment:
We are using the `quarkus extensions add` approach to add dependencies. Then you can proceed and explain what it does. See: https://kiegroup.github.io/kogito-docs/serverlessworkflow/main/cloud/build-workflow-image-with-quarkus-cli.html#_using_a_kogito_example_application

Reply:
I have added as mentioned in the other doc.. Please let me know if that is okay